README.md

```diff
@@ -44,12 +48,20 @@ scons platform=iphone target=debug
 
 ### For Godot 4.x:
 
-```bash
-scons platform=ios target=debug
-```
 
-You don't have to wait for full engine compilation, as header files are generated first.
-Once the actual compilation starts, you can stop this command by pressing <kbd>Ctrl + C</kbd>.
+1. After cloning this repository, go to `godot` submodule and switch to the tag of the engine version you are working with. For example, `git checkout 4.4-stable`.
+
+2. If the version is lower than 4.4, you will need to cherry-pick the commit from [here](https://github.com/godotengine/godot/pull/97510 ).  Depending on the version, it might cause a conflict in the file `drivers/vulkan/rendering_context_driver_vulkan.h` which you resolve by taking the changes from the cherry-picked commit for the conflicting section only.

```

Comment:
```suggestion
2. If the version is lower than 4.4, you will need to cherry-pick the commit from [here](https://github.com/godotengine/godot/pull/97510). Depending on the version, it might cause a conflict in the file `drivers/vulkan/rendering_context_driver_vulkan.h` which you resolve by taking the changes from the cherry-picked commit for the conflicting section only.

```